<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>x-axis</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGenerate">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>z-axis </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>2d-scan</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>3d-scan</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Scan 
length/area</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Resolution/ 
 increments</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>410</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="suffix">
     <string>[mm]</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="suffix">
     <string>[mm]</string>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="suffix">
     <string>[mm]</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="suffix">
     <string>[deg]</string>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>bed</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>440</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="suffix">
     <string>[mm]</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="suffix">
     <string notr="true" comment="s">[deg]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xLength">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string notr="true">0-220</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string notr="true">0-320</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string notr="true">0-360</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xRes">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string notr="true">0-220</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string notr="true">0-220</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string notr="true">0-360</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>[mm]</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>[mm]</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>[mm]</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>[mm]</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>[deg]</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>[deg]</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>measuring time sensor</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="sensorDelay">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>200</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>[ms]</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="xLength_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <double>300.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="xRes_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="sensorDelay_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>370</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionsave_as"/>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionsave_as">
   <property name="text">
    <string>save as</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>checkBox_1</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>doubleSpinBox_3</tabstop>
  <tabstop>doubleSpinBox</tabstop>
  <tabstop>doubleSpinBox_4</tabstop>
  <tabstop>doubleSpinBox_2</tabstop>
  <tabstop>doubleSpinBox_5</tabstop>
  <tabstop>doubleSpinBox_6</tabstop>
  <tabstop>pushButtonGenerate</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEdit_3</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEdit_6</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
